src/main/java/org/folio/Environment.java

```diff
@@ -17,6 +21,18 @@ public static int getScheduledAnonymizationNumberOfLoansToCheck() {
     return getVariable("SCHEDULED_ANONYMIZATION_NUMBER_OF_LOANS_TO_CHECK", 50000);
   }
 
+  public static boolean getCheckOutFeatureFlag() {
+    return Boolean.parseBoolean(getPropertyValue("CHECKOUT_LOCK_FEATURE_ENABLED", "false"));
+  }
+
+  public static List<Integer> getRetryIntervals() {
+    return getIntervalArr(getPropertyValue("RETRY_INTERVAL_MS","300"));
```

Comment:
Should this default value be different from the default value in the environment variables (module descriptor)?

Reply:
This wont create any effect effect when the value of the check_out_feature flag is false. But I changed the value similar to module descriptor.

Reply:
Instead of environment variable now we are using mod-settings to maintain configuration of checkout lock feature, Hence resolving this comment.